specification/ingest/simulate/SimulatePipelineRequest.ts

```diff
@@ -43,10 +44,14 @@ export interface Request extends RequestBase {
     verbose?: boolean
   }
   body: {
+    /**
+     * JSON body for the document.
+     */
+    _source?: UserDefinedValue
```

Comment:
The docs suggest that this should be in `Document`. But maybe the server code supports passing the source of a single document? But then how does that interact with `docs`?

Reply:
not sure... I did some tests and the server just seems to ignore it. I'll ask around

Reply:
so this is another case where the parser will accept anything, but the only fields that actually get processed are `docs` and `pipeline`, so I removed `_source`